### PR TITLE
ENT-9094: Added notes about excluding cf_promises* generated files from VCS

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: ENT-9094
Changelog: None

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/235/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/235/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-235/output/_site/

**Comment by nickanderson** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/236/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/236/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-236/output/_site/

File: web-ui/hub_administration/policy-deployment.markdown [hunk 1]
```diff
@@ -26,12 +26,19 @@ the contents of `masterfiles` to it with the following commands (assuming you
 are already in your local repository checkout):
 
 ```
+cat <<EOF >> .gitignore
+cf_promises_validated
+cf_promises_release_id
+EOF
```
Comment (olehermanse) [inline web-ui/hub_administration/policy-deployment.markdown:32]: Here I would prefer the simpler command like above;

```
echo cf_promises_validated >> .gitignore
echo cf_promises_release_id >> .gitignore
```

**Pull request merged**